heartbeat.py

```diff
@@ -1,22 +1,26 @@
 #!/usr/bin/env python3
-import datetime
 import hashlib
 import json
 import time
 import yaml
+from pathlib import Path
+import smtplib
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
 
 DATETIME_FORMAT = '%m/%d %H:%M'
 
+# without changing to string only works after python 3.5
+f_path = str(Path(__file__).parent) + '/'
```

Comment:
What's the idea behind putting the config relative to the script? In my setup I just set the working directory in the systemd timer.

Reply:
Hey thanks for reviewing!

I run this heartbeat on crontab like written [here](https://github.com/wadetb/heartbeat#installation). And crontab [runs the script in the home directory](https://unix.stackexchange.com/a/38956), if we don't want to load the config in a relative way, we have to add `cd path/to/heartbeat/folder` in the crontab script. I would suggest it's more reliable to setup a relative loading feature

Reply:
How about an optional command line argument to set the `f_path` but it defaults to the current dir? So as not to break any existing users.

Reply:
updated, please have a look!